ddtrace/contrib/azure.functions/patch.py

```diff
@@ -0,0 +1,39 @@
+from ddtrace import config
+from ddtrace.internal.schema import schematize_service_name
+import azure.functions
+
+from ...internal.utils.wrappers import unwrap
+from ddtrace.vendor import wrapt
+config._add(
+    "azure.functions",
+    {
+        "distributed_tracing": True,
+        "_default_service": schematize_service_name("azure.functions"),
+    },
+)
+
+
+def get_version():
+    return ""
```

Comment:
We need to implement this function. This should return the version of the azure function package